skills/ad4m/SKILL.md

```
@@ -175,15 +177,28 @@ ad4m-executor run --app-data-path ~/.ad4m-server \
+```bash
+curl -s http://localhost:12100/graphql \
+  -H "Content-Type: application/json" \
+  -H "Authorization: <admin-credential>" \
+  -d '{"query":"mutation { agentUnlock(passphrase: \\"<passphrase>\\", holochain: true) { isUnlocked } }"}'
+```
+
+Without this, `request_capability` and `generate_jwt` return `"Wallet is locked"`.
+
+**⚠️ MCP sessions require a two-step handshake.** After the `initialize` request, you **must** send a `notifications/initialized` notification before any tool calls. Without it, the session terminates on the next request. See `skills/ad4m/references/mcp.md` for the full handshake sequence.
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)